<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>630</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate sh(x)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>661</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Answer: (press calculate)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton_2">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>210</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate ch(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate th(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string> Calculate cth(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>160</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide notes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showdialog">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>470</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>open dialogwindow</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>element1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>element2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>element3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>element4</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1227</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>1258</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
